nightwatch/tests/songs/songAction.ts

```diff
@@ -4,272 +4,273 @@
-//                 const songId = tdContentId.value;
-//                 browser
-//                     .getText('#render-table > div > div > div > table > tbody > tr:nth-child(1) > td:nth-child(3) a', async (tdContentName) => {
-//                         await CommonFunction.deleteBySongId(browser, tdContentId.value, tdContentName.value);
-//                     })
-//             })
-//             .pause(CONSTANTS.PAUSE_TIMEOUT)
-//             // delete artist as well
-//             .url(CONSTANTS.ARTIST.BASE_URL + "?q=" + artistName)
-//             .pause(CONSTANTS.PAUSE_TIMEOUT)
-//             .waitForElementVisible('#modal-title-ajax', CONSTANTS.WAIT_FOR_ELEMENT_VISIBLE_TIMEOUT)
-//             .pause(CONSTANTS.PAUSE_TIMEOUT)
-//             .assert.containsText('#modal-title-ajax', 'Artist', "Testing if heading is Artist")
+//   //     // get id and delete that one
+//   //     .getText('#render-table > div > div > div > table > tbody > tr:nth-child(1) > td:nth-child(1)', async (tdContentId) => {
+//   //       console.log('artist:', artistName, tdContentId)
+//   //       const songId = tdContentId.value;
+//   //       browser
+//   //         .getText('#render-table > div > div > div > table > tbody > tr:nth-child(1) > td:nth-child(3) a', async (tdContentName) => {
+//   //           await CommonFunction.deleteBySongId(browser, tdContentId.value, tdContentName.value);
+//   //         })
+//   //     })
+//   //     .pause(CONSTANTS.PAUSE_TIMEOUT)
+//   //     // delete artist as well
+//   //     .url(CONSTANTS.ARTIST.BASE_URL + "?q=" + artistName)
+//   //     .pause(CONSTANTS.PAUSE_TIMEOUT)
+//   //     .waitForElementVisible('#modal-title-ajax', CONSTANTS.WAIT_FOR_ELEMENT_VISIBLE_TIMEOUT)
+//   //     .pause(CONSTANTS.PAUSE_TIMEOUT)
+//   //     .assert.containsText('#modal-title-ajax', 'Artist', "Testing if heading is Artist")
 
-//             .getText('#render-table > div > div > div > table > tbody > tr > td:nth-child(1)', async (tdContentId) => {
-//                 const groupId = tdContentId.value;
-//                 browser
-//                     .getText('#render-table > div > div > div > table > tbody > tr:last-child > td:nth-child(2) > a', async (tdContentName) => {
-//                         await CommonFunction.deleteByArtistId(browser, tdContentId.value, tdContentName.value);
-//                     })
-//             })
-//             .end();
-//     }
+//   //     .getText('#render-table > div > div > div > table > tbody > tr > td:nth-child(1)', async (tdContentId) => {
+//   //       const groupId = tdContentId.value;
+//   //       browser
+//   //         .getText('#render-table > div > div > div > table > tbody > tr:last-child > td:nth-child(2) > a', async (tdContentName) => {
+//   //           await CommonFunction.deleteByArtistId(browser, tdContentId.value, tdContentName.value);
+//   //         })
+//   //     })
+//   //     .end();
+//   // }
```

Comment:
why comment again here ?

Reply:
need to work on this file, because test cases are running fine on a local machine but failing on sauce-lab.